templates/memcached.conf.j2

```diff
@@ -20,7 +20,7 @@ logfile /var/log/memcached.log
 # Start with a cap of 64 megs of memory. It's reasonable, and the daemon default
 # Note that the daemon will grow to this size, but does not start out holding this much
 # memory
--m {{ memcached_memory|default('64') }}
+-m {{ memcached_memory|default('128') }}
```

Comment:
you should be able to do this in your playbook so that the change only affects TES